app/server/models/user.py

```diff
@@ -570,40 +571,20 @@ def get_most_relevant_transfer_usage(self):
         '''
 
         sql = text('''
-            SELECT *, COUNT(*)  FROM
-                (SELECT u.business_usage_id FROM credit_transfer c
-                LEFT JOIN "user" u ON c.recipient_user_id = u.id
-                WHERE sender_user_id = {} AND c.transfer_status = 'COMPLETE'
+            SELECT *, COUNT(*) FROM
+                (SELECT c.transfer_use::text FROM credit_transfer c
```

Comment:
we want to group by transfer use instead of business_usage_id since business_usage_id is optional on users, but transfer use will always be set